VCNetworking/VCNetworking/decoders/BasicServiceResponseDecoder.swift

```diff
@@ -0,0 +1,12 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.txt in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+
+import Foundation
+
+public struct BasicServiceResponseDecoder: Decoding {
+    public func decode(data: Data) throws -> String? {
+        return nil
```

Comment:
Does it ignore the body? In that case I would probably name it something like NilServiceResponseDecoder? Or can we make the place where it's used an optional and not pass one?